orchestrator/gravity_abi/src/erc20.rs

```diff
@@ -1,5 +1,6 @@
 pub use erc20::*;
 #[allow(clippy::too_many_arguments, non_camel_case_types)]
+#[allow(clippy::module_inception)]
 pub mod erc20 {
```

Comment:
should this internal module be made private, since it all goes exported in the parent one anyway?